fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalBinary.java

```diff
@@ -43,4 +46,16 @@ public LogicalBinary(PlanType type, Optional<GroupExpression> groupExpression,
     }
 
     public abstract List<Slot> computeOutput();
+
+    @Override
+    public JSONObject toJson() {
+        JSONObject json = new JSONObject();
+        json.put("PlanType", getType().toString());
+        JSONArray childrenJson = new JSONArray();
+        for (Plan child : children) {
+            childrenJson.put(child.toJson());
+        }
+        json.put("children", childrenJson);
+        return json;
+    }
```

Comment:
we don't implement this func for `Unary` `Binary`.
We can implement it for `AbstractPlan`, and check children().size is enough

Reply:
I considered this implementation, but found leafPlan do not have children member, so this can not be done